scripts/UpdateDependencies.ps1

```diff
@@ -62,6 +66,21 @@ foreach ($varName in ($variables.Keys | sort)) {
 if ($count -gt 0) {
     Write-Host -f Cyan "Updating $count version variables in $depsPath"
     SaveXml $dependencies $depsPath
+
+    # Ensure dotnet is installed
+    Invoke-Expression -Command "$PSScriptRoot\..\run.ps1 install-tools"
+
+    $ProjectPath = "$PSScriptRoot\update-dependencies\update-dependencies.csproj"
+
+    # Restore the app
+    Write-Host "Restoring App $ProjectPath..."
+    Invoke-Expression "dotnet restore `"$ProjectPath`""
```

Comment:
dotnet-run runs dotnet restore, so you can condense this do just `& dotnet run -p $ProjectPath @EnvVars`